governance.md

```diff
@@ -1,27 +1,30 @@
 ---
 layout: contribute
-title: Governance
+title: Contribution policy
 ---
 
-# Governance
+# Contribution policy
 
-The Bazel project is led by a core group of contributors, initially Googlers, and managed by the
-community. The group of core contributors is self-managing - core contributors are added by two
+This page covers Bazel's governance model and contribution policy.
+
+## Governance model
+
+The Bazel project is led by a core group of contributors,  and managed by the
```

Comment:
Delete the comma and one of the spaces after contributors

Reply:
Done.